closure/compiler/closure_js_library.bzl

```diff
@@ -74,8 +83,9 @@ closure_js_library = rule(
         "deps": JS_DEPS_ATTR,
         "exports": JS_DEPS_ATTR,
         "language": attr.string(default=JS_LANGUAGE_DEFAULT),
+        "suppress": attr.string_list(),
```

Comment:
Does it make sense to add `suppress` support to `closure_js_binary()` for symmetry in the API?